test/e2e/gardener/managedseed/managedseed.go

```diff
@@ -0,0 +1,113 @@
+// SPDX-FileCopyrightText: SAP SE or an SAP affiliate company and Gardener contributors
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package managedseed
+
+import (
+	"time"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	gardencorev1beta1 "github.com/gardener/gardener/pkg/apis/core/v1beta1"
+	"github.com/gardener/gardener/pkg/utils/kubernetes/health"
+	. "github.com/gardener/gardener/pkg/utils/test/matchers"
+	. "github.com/gardener/gardener/test/e2e/gardener"
+)
+
+// ItShouldCreateManagedSeed creates the managedseed object
+func ItShouldCreateManagedSeed(s *ManagedSeedContext) {
+	GinkgoHelper()
+
+	It("Create ManagedSeed", func(ctx SpecContext) {
+		s.Log.Info("Creating ManagedSeed")
+
+		Eventually(ctx, func() error {
+			if err := s.GardenClient.Create(ctx, s.ManagedSeed); !apierrors.IsAlreadyExists(err) {
+				return err
+			}
+			return StopTrying("managedseed already exists")
+		}).Should(Succeed())
+	}, SpecTimeout(time.Minute))
+}
+
+// ItShouldWaitForManagedSeedToBeReady waits for the ManagedSeed to be ready
+func ItShouldWaitForManagedSeedToBeReady(s *ManagedSeedContext) {
+	GinkgoHelper()
+
+	It("Should wait for ManagedSeed to be ready", func(ctx SpecContext) {
+		Eventually(ctx, func(g Gomega) {
+			g.Expect(s.GardenClient.Get(ctx, client.ObjectKeyFromObject(s.ManagedSeed), s.ManagedSeed)).To(Succeed())
+			g.Expect(health.CheckManagedSeed(s.ManagedSeed)).To(Succeed())
+		}).Should(Succeed())
+	}, SpecTimeout(5*time.Minute))
```

Comment:
If I see it correctly, this step had a timeout of `10m` before.
```suggestion
	}, SpecTimeout(10*time.Minute))
```